examples/bri-3/src/bri/zkp/agents/proofStorage.agent.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { BlockchainService } from '../../../../blockchain/blockchain.service';
+
+import { Proof } from '../models/proof';
+
+@Injectable()
+export class ProofStorageAgent extends BlockchainService {
+  async storeProofInShieldContract(proof: Proof): Promise<void> {
```

Comment:
i am not sure at all regarding this question, but just to double check, do we need to store some info here in db too?

Reply:
we only need to store the proof in the DB @skosito, we are storing the proof onchain as soon as we verify it onchain ... why use expensive onchain storage if the proof is in the transaction payload anyway

Reply:
@Therecanbeonlyone1969 i did not refer to using expensive onchain storage in this comment, i asked about what if anything do we store in db

Reply:
In our implementation, the proof (which includes id, ownerAccountId, payload(content-addressable hash of document), signature) is not being stored anywhere currently. But it is possible to store it in the DB. Currently, we are only storing the payload of the proof (content-addressable hash of document) onchain so as to allow participants to access this information and sync their own data accordingly.